### PR TITLE
Add 이유찬 정보 in team-D-description.md

### PR DESCRIPTION
**Pull request by youngjun0627**
## Description
Team D 팀원 이유찬 정보 추가

## Context
1주차 과제

- 환경설정 및 Git/GitHub 을 통한 피드백 및 PR 규칙 익히기

## Checklist:
- [ ] Updated tests for this change.
- [ ] Tested the changes locally.
- [ ] Updated documentation.

### CODE REVIEW

File: team-D-description.md [hunk 1]
```diff
@@ -9,8 +9,10 @@
...
-
-
 이름 : 백종현   
-관심 컴포넌트 : backend.ai-manager, backend.ai-webserver
-간단 자기소개 : 안녕하세요. 올해 2월에 졸업하고 취업을 하게되어 개발자로 일하고 있는 백종현입니다. 학생때는 간단한 개발만 해왔고 일한지 6개월 밖에 되지 않기도 했기 때문에, 스스로 "개발자"라고 부르는 것을 부끄러워 하는 사람입니다. 스스로 개발자라고 당당하게 말할 수 있는 사람이 되는 것이 목표입니다. 현재 개발한지 오래된 프로젝트에서 유지보수성 개발을 하고 있기 때문에, 주로 레거시 기반의 자바 코드를 작성하고 있습니다. 이번 기회를 통해 회사일에서 벗어나 파이썬 개발 등 다양한 경험을 해보고 싶습니다. 인상이 좋은 편이 아니라 다가가기 어려워 보겠지만, 많이 열려있는 사람이니 편하게 대해주셨으면 좋겠습니다:)
+관심 컴포넌트 : backend.ai-manager, backend.ai-webserver   
+간단 자기소개 : 안녕하세요. 올해 2월에 졸업하고 취업을 하게되어 개발자로 일하고 있는 백종현입니다. 학생때는 간단한 개발만 해왔고 일한지 6개월 밖에 되지 않기도 했기 때문에, 스스로 "개발자"라고 부르는 것을 부끄러워 하는 사람입니다. 스스로 개발자라고 당당하게 말할 수 있는 사람이 되는 것이 목표입니다. 현재 개발한지 오래된 프로젝트에서 유지보수성 개발을 하고 있기 때문에, 주로 레거시 기반의 자바 코드를 작성하고 있습니다. 이번 기회를 통해 회사일에서 벗어나 파이썬 개발 등 다양한 경험을 해보고 싶습니다. 인상이 좋은 편이 아니라 다가가기 어려워 보겠지만, 많이 열려있는 사람이니 편하게 대해주셨으면 좋겠습니다:)   
+
+이름 : 이유찬   
+관심 컴포넌트 : backend.ai.webserver, backend.ai.storage-proxy, backend.ai.agent, backend.ai.manager
```
Comment (lizable) [inline team-D-description.md:17]: Please update the name of some components, such as `backend.ai.agent` and `backend.ai.manager`

**Review by lizable — CHANGES_REQUESTED** [review]
Thx for the contribution @youngjun0627.
Btw, I found a minor typo error. Please see the comment and fix it.

File: team-D-description.md (hunk 1 above)
Comment (youngjun0627) [reply team-D-description.md]: I've fixed it!!

**Review by lizable — APPROVED** [review]
LGTM.

**Pull request merged**